cdap-app-fabric/src/main/java/io/cdap/cdap/metadata/LineageHTTPHandler.java

```diff
@@ -121,6 +121,22 @@ public void datasetFields(HttpRequest request, HttpResponder responder,
     }
   }
 
+  @GET
+  @Path("/namespaces/{namespace-id}/datasets/{dataset-id}/lineage/allfields")
```

Comment:
How is this endpoint supposed to be different that the `/fields` one? Having the name `allfields` is not exactly conform to our existing REST structure. I don't see javadoc on handler methods in this class, and not all of them are obvious, hence I ask.

Reply:
`/fields` endpoint is to provide information on lineage. `allfields` is to provide field level lineage. It is unclear, I will add the javadoc